docs/staticConfiguration.md

```diff
@@ -6,7 +6,7 @@ Static Configuration Guide
 * [Configuration](#overview)
 * [Usage](#overview)
```

Comment:
Although this is not part of the PR, could you please update the links of the index?